recipes/rasam/build.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+set -e
+
+BUILD_DIR="build"
+
+mkdir -p ${BUILD_DIR}
+cd ${BUILD_DIR}
+
+# Create build files
+cmake .. -DCMAKE_INSTALL_PREFIX=$PREFIX -DCMAKE_BUILD_TYPE=Release
```

Comment:
I'd recommend to also pass `${CMAKE_ARGS}` to define some conda-forge specific variables.

Reply:
Done